### ISSUE TITLE
Feature Request: changing how the Webhook handles DM based Database for users

### ISSUE THREAD

**Issue by Batlh** [versx/WhMgr-UI#58]
I request a option so that the personal DM System can be controlled from the user itself so that if people don't like PVP stats it can be deactivated from themselves over the ui. Unlike now where only it is possible to change it for every user not individual.
It doesn't make any sense we can Programm the bot for DM with personal location area and co but we can't deactivate features some don't want to use or don't need.
It would be great to have it implemented also it shouldn't be that much work to that change.

**Comment by versx** [comment]
> I request a option so that the personal DM System can be controlled from the user itself so that if people don't like PVP stats it can be deactivated from themselves over the ui. Unlike now where only it is possible to change it for every user not individual.
> It doesn't make any sense we can Programm the bot for DM with personal location area and co but we can't deactivate features some don't want to use or don't need.
> It would be great to have it implemented also it shouldn't be that much work to that change.

I agree _some_ parts should be modularized based on the user's input, which I can implement eventually, but, with this kind of statement: `also it shouldn't be that much work to that change.` I'd like to suggest that I'm waiting on your PR since it's not much work. 😄 
It's not much work to just add a PvP stats specific toggle for the user, but having in mind expansion, what if the user doesn't want to see any Catch Rates (which are useless anyways), Typings, Size, Moves, etc?

Maybe the solution is really just allow the user to specify from either a list of templates of their DM's or possibly full control how they look? More work, but definitely more beneficial imo.
